notebooks/1.2-ijd-fetch-circle-elevations.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>Is there a reason this data_iterator was added outside of the loop below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>It looks like I probably added that cell as a test and should be removed. I removed it.</p>